### DISCUSSION TITLE
conecction.count

### DISCUSSION THREAD

**Discussion by LeonardoRaimondo** [ujjwalguptaofficial/JsStore#252] [General]
hi!

my query is, how i get the results from connection.count.
Because it tells me that it is a promise.

async function countUser(){
    const results = await connection.count({
        from: "users"
    });
    return results
}
console.log(countUser());

in google inspector I get 
Promise {<pending>}
[[Prototype]]: Promise
[[PromiseState]]: "fulfilled"
[[PromiseResult]]: 0

**Comment by LeonardoRaimondo** [comment]
thanks for the help and excellent tool.
Regards 
Leonardo

**Comment by LeonardoRaimondo** [comment]
hi!
Thanks, I already solved the problem.
My mistake is due to lack of knowledge.
want to read the data outside of the asynchronous method.

Cheers
Leonardo

**Reply by ujjwalguptaofficial** [reply]
I am glad that you were able to found the solution by yourself :) .
